client/src/components/SubmissionTabs/SubmissionTabs.tsx

```diff
@@ -46,18 +61,26 @@ function SubmissionTabs({ card }: cardProps): JSX.Element {
         variant="fullWidth"
         onChange={handleChange}
       >
-        <Tab label="Designs (30)" />
+        <Tab label={countImages()} />
         <Tab label="Brief" />
       </Tabs>
       <Paper elevation={1}>
         <Box p={3}>
           <SubmissionTabPanel value={value} index={0}>
             <Grid container spacing={5}>
-              {card.map((data, key) => (
-                <Grid item md={3} xs={12} key={key}>
-                  <SubmissionCard imageSrc={data.imageFiles[0]} author={data.userID} />
-                </Grid>
-              ))}
+              {card.map((data, key) =>
+                data.imageFiles.length > 1 ? (
+                  data.imageFiles.map((image) => (
+                    <Grid item md={3} xs={12} key={key}>
```

Comment:
Please avoid the usage of indexes as keys, is there something unique we can use here?

Ref: https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9

Reply:
Okay @rajivtitus ill fix that